src/main/java/core/basesyntax/CircleSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class CircleSupplier {

```

Comment:
The `random` instance variable is declared but never used. Consider removing this unused variable.